powerauth-java-crypto/src/main/java/io/getlime/security/powerauth/crypto/lib/encryptor/ClientEncryptor.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * PowerAuth Crypto Library
+ * Copyright 2023 Wultra s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getlime.security.powerauth.crypto.lib.encryptor;
+
+import io.getlime.security.powerauth.crypto.lib.encryptor.exception.EncryptorException;
+import io.getlime.security.powerauth.crypto.lib.encryptor.model.*;
+
+/**
+ * The {@code ClientEncryptor} interface provides End-To-End Encryption for PowerAuth Clients.
+ */
+public interface ClientEncryptor {
+
+    /**
+     * Get this encryptor's identifier.
+     * @return This encryptor's identifier.
+     */
+    EncryptorId getEncryptorId();
+
+    /**
+     * Get parameters used to construct this encryptor.
+     * @return EncryptorParameters used to construct this encryptor.
+     */
+    EncryptorParameters getParameters();
```

Comment:
`ServerEncryptor` uses `getEncryptorParameters` method name, perhaps we should unify these method names. The order of methods is also a litle bit different between these two interfaces.